manifests/Logitech/Options/8.20.329.yaml

```diff
@@ -0,0 +1,26 @@
+# ================== General ==================
+Id: Logitech.Options
+Publisher: Logitech
+Name: Logitech Options
+Author: Logitech
+Description: Logitech Options is a powerful and easy-to-use application that enhances your Logitech mice, keyboards, and touchpads. Customize your device with Options and do more than you thought possible.
+AppMoniker: logitechoptions
+Tags: mouse,logitech,logi
+#FileExtensions:
+#Protocols: 
+
+# ================== License ==================
+Homepage: https://www.logitech.com/it-it/product/options
+License: copyright (c) logitech
+LicenseUrl: https://secure.logitech.com/en-us/footer/terms-of-use
+
+# ================= Installer =================
+Version: 8.20.329
```

Comment:
The Version defined here and in the title are different from version in the download link. It should be 8.36.86.